site/content/integrate/examples/_index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Examples"
+date: "2017-01-19T12:01:23-04:00"
+section: "integrate"
+---
+
+# Examples
```

Comment:
I'll try to look at other integrations we can highlight. Another one that comes to mind is https://github.com/mattermost/mattermost-bot-sample-golang

Reply:
Actually, these integrations are good examples. Only other one that comes to mind is the Golang bot https://github.com/mattermost/mattermost-bot-sample-golang

I'm wondering if we should make the title more descriptive, e.g. `Hubot adapter for Mattermost` and reference the project name and link in the description. Something like `MattermostOnFire` isn't obvious on first ready for instance. Thoughts?

Reply:
Yeah, agreed titles could be better. Feel free to update them if you want (if I don't get to it first)